### ISSUE TITLE
More complete test fixture for gatsby-source-plone

### ISSUE THREAD

**Issue by datakurre** [collective/gatsby-source-plone#61]
@polyester Paul, as a stakeholder, would you be able to help by composing a more complete test fixture in Plone? With the test fixture I mean example set of Plone content to use and test with the gatsby plugin.

Here is how updating the fixture should currently work:

Requirements:

* bash
* curl
* make
* docker
* docker-compose
* jq https://stedolan.github.io/jq/

Clone the repository and launch our backend Plone with the current fixture:

```bash
git clone git@github.com:collective/gatsby-source-plone
cd gatsby-source-plone
make init-backend
```

Now there should be a Plone with admin:admin credentials running at http://localhost:8080/Plone/

Create content in Plone

Export the content by

```bash
make export-fixture
```

Add and commit the exported JSON files with git and push. CI will import the exported JSON files and update gh-pages from it (once merged into master).

**Comment by polyester** [comment]
Can do, although traveling with reduced connectivity until sunday.

Shall I make a  site with 
- easy subfolder (nice, ascii text, just page/image/news)
- medium subfolder (Unicode text, PDF file)
- hard subfolder (a collection to order news, a simple custom content type, maybe even emojji in text/titles)

so we can examine how gatsby handles those cases progressively and not test all the difficult stuff at the same time? Or does that not make sense?

**Comment by datakurre** [comment]
Travelling sounds like an excellent time to do that on local Plone :) (started with make init-backend or make start) Just try it first on good connection so that everything gets downloaded

Hard style folder would be best. We already have the easy one.

Make commands are listed in readme. init-backend always empties Plone at first (zope undo works). make start just starts the Plone. After you are done with make export, make purge removes everything (including the docker container).

**Comment by datakurre** [comment]
Oh, “travelling until sunday”.

So, no hurry then. The docker image is huge so better to not download it on mobile.

**Comment by datakurre** [comment]
And let me soften the folder a bit. We can not import custom content types yet :( So “hard”, but with only default types.

**Comment by datakurre** [comment]
Remember that we currently build https://collective.github.io/gatsby-source-plone/ from the test fixture. It adds some extra challenge here, but encourages to test with "real data", "eat our own dogfood" and write nice site for our Gatsby integration. (Also ``./docs`` are included, probably moved into ``docs`` subsection of the test site later.)

We cannot  test custom content types yet, because we cannot import them into our test environment, because Plone REST API does not support importing types yet. 

Collections are supported by the test fixture. Obviously translating them into graphql in gatsby takes effort.

**Comment by datakurre** [comment]
Fixed by #63
